src/js/components/reply-actions.jsx

```diff
@@ -0,0 +1,97 @@
+import React, { Component, PropTypes } from 'react';
+import { connect } from 'react-redux';
+import bindAll from 'lodash.bindall';
+
+import { sendMessage, sendLocation } from '../services/conversation';
+import { getRGB, rgbToHsl } from '../utils/colors';
+import { Location } from './location-icon';
+
+export class ReplyActionsComponent extends Component {
+    static propTypes = {
+        accentColor: PropTypes.string,
+        isAccentColorDark: PropTypes.bool,
+        choices: PropTypes.array.isRequired,
+        locationNotSupportedText: PropTypes.string.isRequired
+    };
+
+    constructor(...args) {
+        super(...args);
+        bindAll(this, 'onReplyClick');
+
+        this.state = {};
+    }
+
+    onReplyClick({text}) {
+        const {dispatch} = this.props;
+        dispatch(sendMessage(text));
+    }
+
+    onSendLocation() {
+        const {dispatch, locationNotSupportedText} = this.props;
+
+        if ('geolocation' in navigator) {
+            dispatch(sendLocation());
+        } else {
+            alert(locationNotSupportedText);
```

Comment:
It's a literal alert.
Talked about this with JP, there are very few browsers that don't support it.
If the button doesn't appear, what do we put in its place? We could use the fallback text, but it introduces troublesome workaround code.